enterprise/fulltext-addon/src/test/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomBackupIT.java

```diff
@@ -55,6 +56,8 @@
 {
```

Comment:
Is this bloom specific or could it be renamed to fulltext?

Reply:
Bloom name makes sense, as it uses the bloom surface.